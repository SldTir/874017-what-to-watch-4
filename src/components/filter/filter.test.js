import React from 'react';
import renderer from 'react-test-renderer';
import Filter from './filter.jsx';

const genre = `All genres`;
const activeClass = `catalog__genres-item catalog__genres-item--active`;

const onFilterButtonClick = () => {};

describe(`ChecksTheCreationOfAFilter`, () => {
  it(`CreatesAFilter`, () => {
    const tree = renderer
      .create(<Filter
        filmGenre={genre}
        activeClass={activeClass}
        onFilterButtonClick={onFilterButtonClick}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
