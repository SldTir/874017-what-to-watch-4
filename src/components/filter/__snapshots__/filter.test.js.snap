// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChecksTheCreationOfAFilter CreatesAFilter 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
  onClick={[Function]}
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    All genres
  </a>
</li>
`;
